src/sinks/vector/config.rs

```diff
@@ -6,31 +6,35 @@ use tonic::body::BoxBody;
 use tower::ServiceBuilder;
 use vector_config::configurable_component;
 
+use super::{
+    service::{VectorResponse, VectorService},
+    sink::VectorSink,
+    VectorSinkError,
+};
 use crate::{
     config::{
-        AcknowledgementsConfig, GenerateConfig, ProxyConfig, SinkContext, SinkHealthcheckOptions,
+        AcknowledgementsConfig, GenerateConfig, Input, ProxyConfig, SinkConfig, SinkContext,
+        SinkHealthcheckOptions,
     },
     proto::vector as proto,
     sinks::{
         util::{
             retries::RetryLogic, BatchConfig, RealtimeEventBasedDefaultBatchSettings,
             ServiceBuilderExt, TowerRequestConfig,
         },
-        vector::v2::{
-            service::{VectorResponse, VectorService},
-            sink::VectorSink,
-            VectorSinkError,
-        },
         Healthcheck, VectorSink as VectorSinkType,
     },
     tls::{tls_connector_builder, MaybeTlsSettings, TlsEnableableConfig},
 };
 
-/// Configuration for version two of the `vector` sink.
-#[configurable_component]
+/// Configurable for the `vector` sink.
```

Comment:
```suggestion
/// Configuration for the `vector` sink.
```

This is what is rendered in the schema (and eventually the docs), right? Seems like we should ensure the doc comments make sense to users.

Reply:
Uh, yeah, oops, that was copy/paste from the v2 config, and I didn't check the wording. Fixed in b63d00912b67291652f48f8f2999357a05bb810e